src/api/payment-requests/payment-requests.md

```diff
@@ -109,11 +108,12 @@ A line item price can be negative to represents a discount.
 
 {% h4 Optional Fields %}
 
-| Field     | Type               | Description                           |
-|-----------|--------------------|---------------------------------------|
-| tax       | {% dt BigNumber %} | Tax rate (percentage).                |
-| discount  | {% dt BigNumber %} | Discount amount in cents.             |
-| productId | String             | Manufacturer's product identifier (eg GTIN/EAN). |
+| Field      | Type               | Description                                         |
+|------------|--------------------|-----------------------------------------------------|
+| tax        | {% dt BigNumber %} | Tax rate (percentage).                              |
+| discount   | {% dt BigNumber %} | Discount amount in cents.                           |
```

Comment:
When price is negative, the `discount` field should not be present as the whole line item represents a discount. Could we specify this as an error case?